app/controllers/users_controller.rb

```diff
@@ -1,10 +1,19 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
-  before_action :find_user, except: %i[import index]
+  before_action :find_user, except: %i[create import index new]
 
   before_action :restrict_to_staff
 
+  def create
+    @user = User.new
```

Comment:
Does this need to be an instance variable?

Reply:
Also, couldn't you just go `user = User.new user_params; if user.save....`

Reply:
Because line 13 has a `render`.

Reply:
OH